docs/modules/ROOT/pages/usage.adoc

```diff
@@ -78,3 +78,67 @@ servers:
     default:
       config: {}
 ----
+
+=== Storage for data volumes
+
+The OPA Operator currently does not support using https://kubernetes.io/docs/concepts/storage/persistent-volumes[PersistentVolumeClaims] for internal storage.
+
+=== Memory requests
+
+You can request a certain amount of memory for each individual role group as shown below:
+
+[source,yaml]
+----
+servers:
+  roleGroups:
+    default:
+      config:
+        resources:
+          memory:
+            limit: '2Gi'
+----
+
+In this example, each OPA container in the `default` role group will have a maximum of 2 gigabytes of memory. To be more precise, these memory limits apply to the container running OPA but not to any sidecar containers (e.g. the BundleBuilder) that are part of the pod.
+
+A general rule of thumb on how to size the memory for OPA is described https://www.openpolicyagent.org/docs/latest/policy-performance/#resource-utilization[here].
+
+For more details regarding Kubernetes memory requests and limits see: https://kubernetes.io/docs/tasks/configure-pod-container/assign-memory-resource/[Assign Memory Resources to Containers and Pods].
+
+=== CPU requests
+
+Similarly to memory resources, you can also configure CPU limits, as shown below:
+
+[source,yaml]
+----
+servers:
+  roleGroups:
+    default:
+      config:
+        resources:
+          cpu:
+            max: '500m'
+            min: '250m'
+----
+
+=== Defaults
+
+If nothing is specified, the operator will automatically set the following default values for resources:
+
+[source,yaml]
+----
+servers:
+  roleGroups:
+    default:
+      config:
+        resources:
+          requests:
+            cpu: 200m
+            memory: 2Gi
+          limits:
+            cpu: "2"
+            memory: 2Gi
```

Comment:
We keep making this error. There must be a broken copy/paste somewhere else ;)
```suggestion
          cpu:
            min: '200m'
            max: "4"
          memory:
            limit: '2Gi'
```

Reply:
Thanks. Yeah that makes me wonder where else this is still in.